api/query/atoms.go

```diff
@@ -146,6 +149,22 @@ func (c AbstractCount) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
 	}
 }
 
+// AbstractLink is represents the root of a link query, whic matches Metadata URLs
+// to a pattern string; it is independent of test runs.
+type AbstractLink struct {
+	Pattern string
+}
+
+// BindToRuns for AbstractLink is a no-op; it is independent of test runs
+func (l AbstractLink) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
+	var netClient = &http.Client{
+		Timeout: time.Second * 5,
+	}
+	metadata := shared.GetMetadataResponse(runs, netClient, log.StandardLogger())
+
+	return Link{l.Pattern, metadata}
```

Comment:
Nit: I'd strongly suggest you to explicitly initialize the struct (naming all the fields). This is a non-trivial struct (with type embedding) and defined quite far away from here so it'd be brittle / less readable to initialize like this.